cypress/commons/actions/generic/AppBar.js

```diff
@@ -90,4 +110,9 @@ export const AppBar = {
   openHelpMenu,
   openAboutDialog,
   closeAboutDialog,
+  getTechnicalInfoDialog,
+  openTechnicalInfoDialog,
+  getTechnicalInfoSolutionName,
+  getTechnicalInfoSolutionDescription,
+  closeTechnicalInfoDialog,
```

Comment:
Some of the new actions are not exported: `getTechnicalInfoButton`, `getTechnicalInfoDialogCloseButton`